backend/migrations/versions/59ab197fe648_add_backfill_read_table_permissions.py

```diff
@@ -0,0 +1,122 @@
+"""add_backfill_read_table_permissions
```

Comment:
do we need to backfill these permissions?

Reply:
Don't we already assign `DATASET_TABLE_READ` permission on creates of tables and on share approvals? And `DATASET_TABLE_READ` already included `GET_DATASET_TABLE` and `PREVIEW_DATASET_TABLE`?

So were we not already covered with assigning these permissions - we just were not originally checking them? Maybe I am missing something?

Reply:
Yes, I checked in DB. This permissions are already in place